lib/dialects/abstract/connection-manager.js

```diff
@@ -52,7 +52,7 @@ class ConnectionManager {
   onProcessExit() {
     if (this.pool) {
       this.pool.drain(() => {
-        Utils.getLogger().debug(`connection drain via pool due to process exit`);
+        Utils.debug(`connection drain via pool due to process exit`);
```

Comment:
We need to maintain same context of `debug` so it should be a global module. I can directly require `logger` module but we will have to access its methods ( `debug` and `deprecate` ) in similar manner.